mmseg/core/evaluation/metrics.py

```diff
@@ -57,11 +57,11 @@ def intersect_and_union(pred_label,
 
     intersect = pred_label[pred_label == label]
     area_intersect = torch.histc(
-        intersect.float(), bins=(num_classes), min=0, max=num_classes)
+        intersect.float(), bins=num_classes, min=0, max=num_classes - 1)
```

Comment:
We may need to test whether the results are consistent for other models. 
We could make another PR for this modification, we may also include impact range in the new PR.

Reply:
We may merge master and resolve conflicts.